evm2wast.c

```diff
@@ -284,10 +300,10 @@ static char *assemble_jumps(size_t *segments, size_t len)
 	return 0;
 }
 
-static void add_stack_check(char *segment)
+static char *add_stack_check(char *segment)
 {
-	char *check = "";
-	char *template = "";
+	char *check = malloc(1);
+	char *template = malloc(0);
```

Comment:
I am not really fond of `malloc(1)` and `malloc(0)` but `realloc(NULL, size)` is perfectly valid so these can be set to `NULL`

Reply:
can't because below we use `strlen` on them what will cause a segfault